arch/riscv/Kconfig

```diff
@@ -166,7 +166,9 @@ config PMP_POWER_OF_TWO_ALIGNMENT
 endif #RISCV_PMP
```

Comment:
On commit message, I guess it should read "Thus, disable PMP_STACK_GUARD when MULTITHREADING is **not** enabled."

Reply:
@edersondisouza 
It's my carelessness. Thank you for pointing that out.
I corrected the message.